common/djangoapps/student/models.py

```diff
@@ -1377,7 +1377,7 @@ def refundable(self):
 
         # If it is after the refundable cutoff date they should not be refunded.
         refund_cutoff_date = self.refund_cutoff_date()
-        if refund_cutoff_date and datetime.now() > refund_cutoff_date:
+        if refund_cutoff_date and datetime.now(tz=UTC) > refund_cutoff_date:
```

Comment:
You can also use [`datetime.utcnow()`](https://docs.python.org/2/library/datetime.html#datetime.datetime.utcnow).